exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,7 @@
 // Declare your function here
-
+function createLongGreeting(myName, age) {
+    return `Hello, my name is ` + myName + " and I'm 30 years old "
+}
 const greeting = createLongGreeting("Daniel", 30);
 
 console.log(greeting);
```

Comment:
well done ,I have just an suggestion in line 3 you use two way: `  ` and " ". I think its better use one of them.